src/index.html

```diff
@@ -7,12 +7,53 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="/src/styles/index.scss"
```

Comment:
The 'rel' attribute in the link tag should be set to 'stylesheet' to link CSS files correctly. The current value seems to be a path which is incorrect.